### DISCUSSION TITLE
airmaid api change

### DISCUSSION THREAD

**Discussion by hrohwer** [automadcms/discussions#4] [General]
Hi, I just noticed that airmad has changed their api key access, is there an update for airmad?

**Comment by marcantondahmen** [comment]
You mean Airtable, right? Airmad didn't change. Can you quickly elaborate what has changed? As far as I know Airtable doesn't support API keys anymore and you will have to generate a PAT instead.

**Reply by hrohwer** [reply]
Thank you, correct Airtable, for some reason it wont work with the new PAT.

**Reply by hrohwer** [reply]
My apologies, it seems to be a problem with my plugin installation :(

**Reply by marcantondahmen** [reply]
Ah ok, have you been able to solve it? How did you install the package?

**Reply by hrohwer** [reply]
To be honest, I installed your new v2 alpha to test and I could add packages again :)

**Reply by marcantondahmen** [reply]
Nice!
